package-lock.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't think we need this file to be committed?
We could ignore it via `.gitignore` maybe?